lib/utils/metrics.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Graphite = require('graphite');
+const Measured = require('measured');
+
+const REPORT_INTERVAL = 5000;
+const GRAPHITE_HOST = process.env.GRAPHITE_HOST || null;
+const GRAPHITE_PORT = process.env.GRAPHITE_PORT || 2003;
+const ENVIRONMENT_NAME = 'test';
+const METRICS_NAMESPACE = process.env.GRAPHITE_NAMESPACE || 'mozilla.magnet';
+
+const data = Measured.createCollection([METRICS_NAMESPACE, ENVIRONMENT_NAME].join('.'));
+
+const failures = data.counter('graphiteReportingFailures');
+
+data.gauge('memory', () => process.memoryUsage().rss);
+
+if (GRAPHITE_HOST) {
```

Comment:
I'd like to keep this as it is, because it's executed once at the module's top level scope, not in a function, having a return could cause weirdness, and wrapping in an IIFE seems over the top right now.  Unless there is a strong reason to do so?